src/frontend/utils/telemetry/Instrumentation.js

```diff
@@ -1,10 +1,21 @@
-const opentelemetry = require("@opentelemetry/sdk-node")
-const { getNodeAutoInstrumentations } = require("@opentelemetry/auto-instrumentations-node")
-const { OTLPTraceExporter } =  require('@opentelemetry/exporter-trace-otlp-grpc')
+const opentelemetry = require('@opentelemetry/sdk-node');
+const { getNodeAutoInstrumentations } = require('@opentelemetry/auto-instrumentations-node');
+const { OTLPTraceExporter } = require('@opentelemetry/exporter-trace-otlp-grpc');
+const { OTLPMetricExporter } = require('@opentelemetry/exporter-metrics-otlp-grpc');
+const { PeriodicExportingMetricReader } = require('@opentelemetry/sdk-metrics');
 
 const sdk = new opentelemetry.NodeSDK({
   traceExporter: new OTLPTraceExporter(),
-  instrumentations: [ getNodeAutoInstrumentations() ]
-})
+  instrumentations: [
+    getNodeAutoInstrumentations({
+      '@opentelemetry/instrumentation-fs': {
+        enabled: false,
```

Comment:
why are we disabling this auto-instrumentation?

Reply:
This is a great question, so I upgrade all open telemetry dependencies to support the metrics feature. Once I upgraded the node auto instrumentation and tried to execute the next JS app I kept getting an error. It seems that the auto instrumentation fs package is breaking the nodejs functionality. Here's a screenshot:

![Screenshot 2022-12-14 at 9 20 10](https://user-images.githubusercontent.com/11051031/207635514-a5db24f3-1c50-46a9-81d1-216ca6a78451.png)


@puckpuck we might have to create an issue for the auto instrumentation package, it started happening when I upgraded to 0.34.0

Reply:
I'm able to replicate the above scenario as well with the `fs` instrumentation enabled.

I did some digging and found a [related issue](https://github.com/open-telemetry/opentelemetry-js-contrib/issues/1315) to this which hasn't been resolved yet.

I would rather we don't have code specifically to get around SDK version upgrade issues.